### ISSUE TITLE
Issue with NSMutableAttributedString in iOS 10.0 and Xcode 8.

### ISSUE THREAD

**Issue by k-sathireddy** [HeshamMegid/HMSegmentedControl#227]
@HeshamMegid  

```
NSMutableAttributedString *newAttrString = [[NSMutableAttributedString alloc] initWithString:NSLocalizedString(@"NEW",nil)];
    [newAttrString addAttribute:NSFontAttributeName value:[UIFont fontWithName:@"SourceSansPro-Semibold" size:14] range:NSMakeRange(0, newAttrString.length)];

 NSMutableAttributedString *nearByAttrString = [[NSMutableAttributedString alloc] initWithString:NSLocalizedString(@"NEARBY",nil)];
    [nearByAttrString addAttribute:NSFontAttributeName value:[UIFont fontWithName:@"SourceSansPro-Semibold" size:14] range:NSMakeRange(0, nearByAttrString.length)];

    NSMutableAttributedString *popularAttrString = [[NSMutableAttributedString alloc] initWithString:NSLocalizedString(@"POPULAR",nil)];
    [popularAttrString addAttribute:NSFontAttributeName value:[UIFont fontWithName:@"SourceSansPro-Semibold" size:14] range:NSMakeRange(0, popularAttrString.length)];
```

self.categorySegmentedControl = [[HMSegmentedControl alloc] initWithSectionTitles:@[newAttrString, nearByAttrString, popularAttrString]];

Before iOS 10.0, all three segmented control headings are displayed. But after updated to iOS 10.0 (Xcode 8), only "NEARBY" and "POPULAR" headings are displayed. "NEW" is not displaying.

"NEW" is only displaying if we provide some space like this:

NSMutableAttributedString *newAttrString = [[NSMutableAttributedString alloc] initWithString:NSLocalizedString(@"NEW       ",nil)];

(or)

if we increase font size like this:

NSMutableAttributedString *newAttrString = [[NSMutableAttributedString alloc] initWithString:NSLocalizedString(@"NEW",nil)];
 [newAttrString addAttribute:NSFontAttributeName value:[UIFont fontWithName:@"SourceSansPro-Semibold" size:20] range:NSMakeRange(0, newAttrString.length)];

Please solve this issue as soon as possible.

**Comment by cooliean** [comment]
I have the same issue

**Comment by Gongbjie** [comment]
I have the same issue too,i haven't set any segmentedControl.titleTextAttributes,but the titles can't show.
<img width="379" alt="2016-10-25 9 36 17" src="https://cloud.githubusercontent.com/assets/11990912/19670043/bc6d851e-9a96-11e6-86b8-bd640bf0b2e7.png">

**Comment by HeshamMegid** [comment]
This should have been fixed in 1.5.3. Please update and let me know if you're still seeing the same issue.
